pinot-spi/src/main/java/org/apache/pinot/spi/config/table/TableConfig.java

```diff
@@ -388,6 +388,12 @@ public List<String> getUpsertComparisonColumns() {
     return _upsertConfig == null ? null : _upsertConfig.getComparisonColumns();
   }
 
+  @JsonIgnore
+  @Nullable
+  public String getUpsertDeleteRecordColumn() {
```

Comment:
Let's make sure the naming is consistent (either use `deleteRecordColumn` or `deletedRecordColumn`). I personally prefer `deleteRecordColumn` because it is used to delete a record

Reply:
huh.. even I prefer `deleteRecordColumn`. Will fix the ones that deviated from this name.